docs/applications/syncthing.md

```diff
@@ -0,0 +1,35 @@
+# Syncthing: Open Source Continuous File Synchronisation
+
+Homepage: [https://syncthing.net/](https://syncthing.net/)  
+Github: [https://github.com/syncthing/syncthing](https://github.com/syncthing/syncthing)  
+Docker: [https://hub.docker.com/r/syncthing/syncthing](https://hub.docker.com/r/syncthing/syncthing)
+
+Syncthing is a continuous file synchronization program. It synchronizes files
+between two or more computers. It strives to fulfill the goals below in summary.
+Syncthing should be:
+
+1. Safe From Data Loss
+2. Secure Against Attackers
+3. Easy to Use
+4. Automatic
+5. Universally Available
+6. For Individuals
+7. For eveything else see the goals document
+
+## Usage
+
+Set `syncthing_enabled: true` in your `group_vars/all.yml` file.
+
+## Specific Configuration
+
+Open the web interface at :8384 to configure.
+
+The tasks/syncthing.yml sets
+```yaml
+    env:
+      STGUIADDRESS: ""
+```
+which disables admin GUI access from anywhere except localhost. You can either
+remove this (at which point it is visible to your whole network), or use
+Traefik. In either case you should consider setting an admin password in the
+syncthing GUI.
```

Comment:
I'm not a fan of instructions that ask users to edit the `task/*.yml` files - it causes issues when users try to merge updates from master. Additionally, blocking GUI access to everywhere except localhost (once you've removed host-mode networking) will mean that it would need to be localhost from *within* the container...which is not exactly trivial :) My preference is this is removed and in the docs we tell users they need to set a password on first use (as is the case with many other applications)